src/repository/opamVCS.ml

```diff
@@ -125,9 +115,18 @@ module Make (VCS: VCS) = struct
                   OpamStd.String.Set.mem basename fset)
           then OpamFilename.remove f)
         (OpamFilename.rec_files repo_root);
-      OpamLocal.rsync_dirs ~args:["--files-from"; (Lazy.force convert_path) stdout_file]
-        ~exclude_vcdirs:false
-        repo_url repo_root
+      let stdout_file =
+        let f = OpamSystem.temp_file "rsync-files" in
+        let fd = open_out f in
+        List.iter (fun s -> output_string fd s; output_char fd '\n') files;
+        close_out fd;
+        f
+      in
+      let args = [
+        "--files-from"; (Lazy.force convert_path) stdout_file;
+        "--exclude"; "_build"; (* tiny clean: exclude _build directory *)
```

Comment:
I believe rsync supports `.gitignore` files, so a future improvement to this hardcoded _build directory could be adding `--filter=":- .gitignore"` to the command line.